modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/services/SeqexecWebClient.scala

```diff
@@ -72,6 +85,7 @@ object SeqexecWebClient {
   def log(record: LogRecord): Future[Unit] =
     Ajax.post(
       url = s"$baseUrl/log",
-      data = default.write(LogMessage.fromLogRecord(record))
+      responseType = "arraybuffer",
+      data = Pickle.intoBytes(LogMessage.fromLogRecord(record))
     ).map(_.responseText)
 }
```

Comment:
There seems to be some repeated code for setting up the `url`, the `responseType` and then pickling and unpickling. Perhaps a helper function which takes the `url` and whatever it needs to pickle and unpickle would make the code more concise. However, on a second thought, it may not be worth extracting, so please don't take this comment too seriously :)